Content on learners reopening pull requests with requested changes (#816)
Umuzi-org/ACN-syllabus: content/projects/katas/level-1/_index.md

At this point you should be using git from the command line. If you drag and drop to upload your code, you are doing it wrong. Git is amazingly important and now is the time to get used to it!

## Handling requested changes on Pull Requests

> Pull requests are an excellent approach to communicate proposed code changes for review and receive feedback from staff and other learners.This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.

Reviewer: ```suggestion
Pull requests are an excellent approach to communicating proposed code changes for review and receiving feedback from staff and other learners. This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.
```
```suggestion
Pull requests are an excellent approach to communicate proposed code changes for review and receive feedback from staff and other learners.This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.
```

> Pull requests are an excellent approach to communicate proposed code changes for review and receive feedback from staff and other learners.This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.

Reviewer: ```suggestion
Pull requests are an excellent approach to communicating proposed code changes for review and receiving feedback from staff and other learners. This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.
```
```suggestion
Pull requests are an excellent approach to communicate proposed code changes for review and receive feedback from staff and other learners.This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.
```

> While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are request. 

Reviewer: ```suggestion
While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are requested. 
```
```suggestion
While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are request. 
```

> While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are request. 

Reviewer: ```suggestion
While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are requested. 
```
```suggestion
While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are request. 
```

### Do not click Compare & pull request

> After commiting and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.

Reviewer: ```suggestion
After committing and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.
```
```suggestion
After commiting and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.
```

> After commiting and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.

Reviewer: ```suggestion
After committing and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.
```
```suggestion
After commiting and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.
```

Reviewer: In the last part, I suggest giving directions to where the reopen button can be found:
* On the tabs on top of the page click on Pull requests, click on closed, open the closed pull request and use the reopen button at the bottom of the page to reopen the pr (something along these lines)

Reviewer: In the last part, I suggest giving directions to where the reopen button can be found:
* On the tabs on top of the page click on Pull requests, click on closed, open the closed pull request and use the reopen button at the bottom of the page to reopen the pr (something along these lines)

At this point you should be using git from the command line. If you drag and drop to upload your code, you are doing it wrong. Git is amazingly important and now is the time to get used to it!

## Handling requested changes on Pull Requests

Pull requests are an excellent approach to communicating proposed code changes for review and receiving feedback from staff and other learners.This usually leads to dialogues and suggestions in the comments, which may lead to more code changes.

While comments and feedback are useful for enhancing code quality, they can easily be lost. To ensure that data isn't lost and a consistent system of record is created, it is worth noting that you are expected to REOPEN a closed pull request if changes are requested. 

## Do not click Compare & pull request

After committing and pushing your changes and you are ready to get reviewed again, do not click on Compare and pull request. If you do this a new pull request will be created and feedback comments would be lost. Click on the Reopen pull request button instead.

## See instructions below to reopening a closed Pull Request: 

> N.B! The pull request must not be merged, just closed.

Reviewer 2: Not sure what this means, when learners get requested changes they can't merge their own PR's anyway. So is this necessary?
